content/data-reporting/reports-api/v3.3/index.mdx

```diff
@@ -94,6 +94,58 @@ curl "https://api.livechatinc.com/v3.3/reports/chats/agents_chatting_duration?fr
 </Code>
 </Section>
 
+<Section>
+<Text>
+
+### Tags
+
+Shows distribution of tags for chats.
+
+#### Specifics
+
+|                     |                                                       |
+| ------------------- | ----------------------------------------------------- |
+| **Method URL**      | `https://api.livechatinc.com/v3.3/reports/chats/tags` |
+| **HTTP method**     | GET                                                   |
+| **Required scopes** | -                                                     |
+
+#### Request
+
+| Query String          | Required | Notes                                                                                                                                                                   |
+| --------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `to`                  | yes      | Date in the RFC3339 format                                                                                                                                              |
+| `from`                | yes      | Date in the RFC3339 format                                                                                                                                              |
+| `agents`              | no       | Agent emails separated by a comma; if not specified, returns the data for all agents within the license.                                                                |
+| `groups`              | no       | Group IDs separated by a comma                                                                                                                                          |
+| `names`               | no       | Names of tags separated by a comma; when tags=:without: you will get total number of chats without tags, when tags=:with: you will get total number of chats with tags. |
+| `distribution`        | no       | Possible values: `hour`, `day-hours`, `day`, `month`, `year`; defaults to `day`                                                                                         |
+
+#### Response
+
+| Field               | Notes                                                 |
+| ------------------- | ----------------------------------------------------- |
+| `total`             | The total number of chats in the specified date range |
+| `records`           | Contains `distribution` objects, for example, `day`   |
```

Comment:
```suggestion
| `records`           | Contains the `distribution` objects, for example, `day`.   |
```

Reply:
You need to add 'the' because you're talking about particular objects – distribution objects.